src/components/cell-control/multi-choice.jsx

```diff
@@ -0,0 +1,49 @@
+import React, {
+  forwardRef,
+} from 'react';
+import PropTypes from 'prop-types';
+import MultiAutocompleter from './multi-autocompleter.jsx';
+
+const MultiChoice = forwardRef(function MultiChoice(props, ref) {
```

Comment:
Since this is basically a cell control implementation of the same pattern that `CustomFieldInputMultipleChoice` offers, IE tight integration for custom fields, we should probably use the same naming pattern.

Reply:
It is a chore, tbd on what to do: 

https://www.pivotaltracker.com/story/show/181300509